### ISSUE TITLE
Cannot access to the assigned docker container port

### ISSUE THREAD

**Issue by farhadmammadli** [nginx-proxy/docker-letsencrypt-nginx-proxy-companion#764]
Hi!
Read the documentation carefully, but unfortunately could not assign any custom port.
[Basic-usage](https://github.com/nginx-proxy/docker-letsencrypt-nginx-proxy-companion/blob/master/docs/Basic-usage.md)

How to assign port to a container on run command? Traditional "docker run -p 8080:80 ..." syntax does not forward proxy to the app, so cannot access from the https://sub.domain.com:8080. 
However, I can access the my app just by visiting https://sub.domain.com.

**Comment by farhadmammadli** [comment]
@buchdag

**Comment by buchdag** [comment]
Hi.

`nginx-proxy` isn't really meant to work with different ports on a per proxied container basis.

You can publish the `80` and `443` ports of the `nginx-proxy` container to different ports on the host (let's say `8080` and `8443`), but then you won't be able to use the companion as the [http-01](https://letsencrypt.org/docs/challenge-types/#http-01-challenge) challenge used to validate your ownership of the domains you request certificate for won't work over anything else than default `80`/`443` ports, and there isn't anything we can do to bypass this.

**Comment by farhadmammadli** [comment]
Thanks for the reply, so i need second (sub)domain if i want to serve another dockerized web app?

**Comment by buchdag** [comment]
> Thanks for the reply, so I need another (sub)domain if I wanna serve any other dockerized web app?

Yep, that's how nginx-proxy is meant to be used. If want to serve three different dockerized apps on the same host, you'll need three different (sub)domains.

> What about support of path like https://sub.domain.com/app1 or https://sub.domain.com/app2

That's currently not supported.
